deployment/clowdapp.yaml

```diff
@@ -1065,6 +1106,85 @@ objects:
 
 parameters:
 #
+# Ephemeral Secret Values:
+#
+- name: DJANGO_SENTRY_DSN
+  displayName: Django (API) Sentry DSN
+  required: false
+  value: ''
+- name: LISTENER_SENTRY_DSN
+  displayName: Django (Listener) Sentry DSN
+  required: false
+  value: ''
+- name: HOUNDIGRADE_SENTRY_DSN
+  displayName: Houndigrade Sentry DSN
+  required: false
+  value: ''
+- name: DJANGO_SECRET_KEY
+  displayName: Django Secret Key
+  required: true
+  value: 'ZXBoZW1lcmFs'
```

Comment:
I trust that this value isn't _actually_ a real secret since it's repeated several times below.

How would you feel about making this a more obvious placeholder value so if someone is troubleshooting they can catch it with their human eyes at a quick glance?

Reply:
It says something like ephemeral in base64, because it feeds into a `data:` section in a secret above. Because this isn't a real template, and I can't base64 values on the fly, we have to have a base64 encoded value here.

Reply:
also that section can't be `stringData` because otherwise ci.ext builds will have our secrets be in plaintext. Remember that output snippet I shared with you a week or so back? Yeaaa....

Reply:
ohhhhh gotcha! I didn't think about it as base64 when I first saw it. Thanks for the clarification!

```
echo "ZXBoZW1lcmFs" | base64 -D
ephemeral
```